samples/sensor/lm73c/boards/nrf52840dongle_nrf52840.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0 {
+	lm73c@48 {
+		compatible = "ti,lm73c";
```

Comment:
You're missing the binding definition `dts/bindings/sensor/ti,lm73c.yaml`.

Reply:
Will investigate

Reply:
created will be on split PR